Source/OctaneGUI/Controls/TextInput.h

```diff
@@ -181,6 +181,7 @@ class TextInput : public ScrollableViewControl
     bool m_Multiline { false };
     bool m_ReadOnly { false };
     bool m_NumbersOnly { false };
+    uint32_t m_MaxCharacters { 0 };
```

Comment:
Should add a SetMaxCharacters/MaxCharacters set/get function similar to SetNumbersOnly/NumbersOnly.

Reply:
Added in https://github.com/thnom/OctaneGUI/commit/079fb6b516a88c2a4334a7b7518609cf522d5bb9